api/v1/swagger.yaml

```diff
@@ -1174,6 +1174,561 @@ paths:
           $ref: "#/responses/genericError"
 
   # ---------------------------------------------------------------------------
+  /collectors:
+    post:
+      security:
+        - jwt: []
+      tags: [ collectors ]
+      summary: Access restricted to Refocus Collector only.
+      description: Access restricted to Refocus Collector only.
+      operationId: postCollector
+      parameters:
+        -
+          name: queryBody
+          description: Request body.
+          in: body
+          required: true
+          schema:
+            type: object
+            description: Access restricted to Refocus Collector only.
+            properties:
+              name:
+                type: string
+                description: Should be unique.
+              description:
+                type: string
+                maxLength: 4096
+                description: Access restricted to Refocus Collector only.
+              helpEmail:
+                type: string
+                description: Access restricted to Refocus Collector only.
+                maxLength: 254
+              helpUrl:
+                type: string
+                maxLength: 2082
+                description: Access restricted to Refocus Collector only.
+            required:
+              - name
+      responses:
+        201:
+          description: Success.
+          schema:
+            $ref: "#/definitions/CollectorResponse"
+        400:
+          $ref: "#/responses/400"
+        403:
+          $ref: "#/responses/403"
+    get:
+      security:
+        - jwt: []
+      tags: [ collectors ]
+      summary: Find a collector or collectors.
+      description: >-
+        Find a collector or collectors. You may query using field filters with asterisk (*) wildcards.
+        You may also optionally specify sort, limit, offset, and a list of fields to include in the response.
+      parameters:
+        -
+          $ref: "#/parameters/CollectorFieldsParam"
+        - name: sort
+          in: query
+          description: >
+            Specify the sort order using a field name, e.g. '...?sort=name'. Prepend the
+            field name with a minus sign to specify descending order, e.g.
+            '...?sort=-name'.
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - name
+              - status
+              - isRegistered
+              - lastHeartBeat
```

Comment:
lastHeartBeat ==> lastHeartbeat

(the word "heartbeat" is an actual word so it doesn't get camel-cased in the middle of the word)